### PR TITLE
Relatedproducts

### PR DESCRIPTION
**Pull request by acerslee**
Changed up styling on my cards. Added testing for my components.

### CODE REVIEW

File: client/components/relatedproducts/related-product-card.jsx [hunk 1]
```diff
@@ -2,17 +2,17 @@ import React, {useState, useEffect} from 'react';
 import {StarFill} from 'react-bootstrap-icons';
 import Modal from 'react-modal';
 import ModalDetails from './modalDetails.jsx';
-import Rating from '../Overview/Rating.jsx';
 import api from '../../../api.js';
 import {StaticRating} from '../../starRating.jsx';
+import regeneratorRuntime from 'regenerator-runtime';
```
Comment (koalarick) [inline client/components/relatedproducts/related-product-card.jsx:7]: Is this related to testing?

**Review by koalarick — APPROVED** [review]
Should be good, just a reminder to folks that this might require an npm install after you pull down the next version of prod

File: client/components/relatedproducts/related-product-card.jsx (hunk 1 above)
Comment (acerslee) [reply client/components/relatedproducts/related-product-card.jsx:7]: @koalarick yes, it's a work-around that I found because I was initially having async errors when I was running Jest.
